src/test/java/uk/gov/justice/digital/service/DomainRefreshServiceTest.java

```diff
@@ -86,16 +89,51 @@ public void test_incident_domain() throws IOException {
             System.out.println("DomainRefresh::process('" + domain.getName() + "') completed");
         } catch (Exception e) {
             System.out.println("DomainRefresh::process('" + domain.getName() + "') failed");
-            handleError(e);
+            fail();
         }
     }
 
     @Test
     public void test_establishment_domain_insert() throws IOException {
         final String domainOperation = "insert";
+        final String domainTableName = "establishment";
+        final String sourcePath = this.folder.toFile().getAbsolutePath() + "/source";
+        final String targetPath = this.folder.toFile().getAbsolutePath() + "/target";
+        final DomainDefinition domain = getDomain("/sample/domain/establishment.domain.json");
+        final DataStorageService storage = new DataStorageService();
+
+        final Dataset<Row> df_agency_locations = utils.getAgencyLocations(folder);
+        utils.saveDataToDisk(TableInfo.create(sourcePath, "nomis", "agency_locations"),
+                df_agency_locations);
+
+        final Dataset<Row> df_internal_agency_locations = utils.getInternalAgencyLocations(folder);
+        utils.saveDataToDisk(TableInfo.create(sourcePath, "nomis", "agency_internal_locations"),
+                df_internal_agency_locations);
+
+        try {
+            System.out.println("DomainRefresh::process('" + domain.getName() + "') started");
```

Comment:
Remove these printlns. If we need test output use a logger. Test output should be clean.

Reply:
done